bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/handler/SensiboAccountHandler.java

```diff
@@ -204,6 +205,8 @@ protected SensiboModel refreshModel() throws SensiboException {
     private <T> T sendRequest(final Request request, final AbstractRequest req, final Type responseType)
             throws SensiboException {
         try {
+            request.header(HttpHeader.ACCEPT_ENCODING, "gzip");
```

Comment:
```suggestion
            request.header(HttpHeader.ACCEPT_ENCODING, HttpHeaders.Values.GZIP);
```

Out of my head, might have a typo